docs/sql-reference/sql-functions/date-time-functions/next_day.md

```diff
@@ -0,0 +1,62 @@
+# next_day
+
+## Description
+
+Returns the date of the first specified DOW(day of week) that occurs after the input date (DATE, DATETIME).
```

Comment:
If the expr1 is already the same as expr2, what is the result?
For example '2023-04-07' is Fri
what's the result for `next_day('2023-04-07', 'Fri')`

Reply:
If expr1 is already the same as expr2, it will return the date 7 days later.
Currently MySQL has not implemented `next_day`, but Oracle has ([link](https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/NEXT_DAY.html#GUID-01B2CC7A-1A64-4A74-918E-26158C9096F6)). `NEXT_DAY('7-April-2023','Fri')` in Oracle gives the same result as my implementation `next_day('2023-04-07', 'Fri')`, both are '2023-04-14'

Reply:
Thanks for your explanation.